scripts/cluster-install/postgresql_config.yaml

```diff
@@ -0,0 +1,20 @@
+postgresql_server:
+  cert_path: /home/centos/install_cluster/certs/<db_instance_name>_cert.pem
+  key_path: /home/centos/install_cluster/certs/<db_instance_name>_key.pem
+  ca_path: /home/centos/install_cluster/certs/ca.pem
+
```

Comment:
You can also add   ssl_client_verification: true here if we want to enable it (as mentioned earlier in this PR).

Reply:
Same comment as the above.

Reply:
Done.